src/options/hugo-options.js

```diff
@@ -1,7 +1,13 @@
 module.exports = {
 	structure: 'hugo [options]',
 	defaults: {
-		preserved_paths: 'node_modules/'
+		environment_variables: [
+			{
+				key: 'HUGO_CACHEDIR',
+				value: '/usr/local/__site/src/hugo_cache/'
+			}
+		],
+		preserved_paths: 'node_modules/,hugo_cache/'
```

Comment:
We may want to also add the `resources` directory to the default list of preserved paths while we're in the area. That way we also cache generated images/assets too